React/Views/RCTScrollView.m

```diff
@@ -381,6 +381,7 @@ @implementation RCTScrollView
   uint16_t _coalescingKey;
   NSString *_lastEmittedEventName;
   NSHashTable *_scrollListeners;
+  CGFloat _lastNonZeroTranslationAlongAxis;
```

Comment:
Could you add some inline documentation here on what this ivar represents and why we need it?